middlewares/hacker.middleware.js

```diff
@@ -112,12 +117,142 @@ async function downloadResume(req, res, next) {
     }
     next();
 }
+/**
+ * Sends a preset email to a user if a status change occured.
+ * @param {{body: {status?: string}, email: string}} req 
+ * @param {*} res 
+ * @param {(err?:*)=>void} next 
+ */
+function sendStatusUpdateEmail(req, res, next) {
```

Comment:
I have an idea to make this a bit more concise.

Reply:
Lemme know